app/views/admin/teachers/reopen_induction/confirm.html.erb

```diff
@@ -0,0 +1,53 @@
+<%
+  page_data(
+    title: "Reopen induction for #{teacher_full_name(@teacher)}",
+    backlink_href: admin_teacher_path(@teacher)
+  )
+%>
+
+<%= govuk_warning_text do %>
+  Are you sure you want to reopen this induction?<br>
+  This action cannot be undone.
+<% end %>
+
+<p class="govuk-body">
+  The following data will be deleted from the latest induction period,
+  and the ECT's induction status will be updated to "In Progress".
+</p>
+
+<%= govuk_summary_list do |list| %>
+  <%= list.with_row do |row| %>
+    <%= row.with_key { "End date" } %>
+    <%= row.with_value { @last_induction_period.finished_on.to_fs(:govuk_uk) } %>
+  <% end %>
+
+  <%= list.with_row do |row| %>
+    <%= row.with_key { "Number of terms" } %>
+    <%= row.with_value { @last_induction_period.number_of_terms.to_s } %>
+  <% end %>
+
+  <%= list.with_row do |row| %>
+    <%= row.with_key { "Outcome" } %>
+    <%= row.with_value do %>
+      <%= govuk_tag(
+            text: @last_induction_period.outcome.humanize,
+            colour: @last_induction_period.outcome == "pass" ? "green" : "red"
+          ) %>
+    <% end %>
+  <% end %>
+<% end %>
+
+<div class="govuk-button-group">
+  <%= govuk_button_to(
+        "Reopen induction",
+        admin_teacher_reopen_induction_path(@teacher),
+        method: :patch,
+        warning: true
+      ) %>
+  <%= govuk_button_to(
+        "Cancel",
+        admin_teacher_path(@teacher),
+        method: :get,
+        secondary: true
+      ) %>
```

Comment:
Should this be a [govuk_button_link_to](https://govuk-components.netlify.app/helpers/button/#link-styled-like-a-button) instead of a button with `method: :get`?